app/frontend/src/features/dashboard/Home.tsx

```diff
@@ -0,0 +1,151 @@
+
+  return (
+    <>
+      <PageTitle>{WELCOME}</PageTitle>
+      <DashboardBanners />
+      {process.env.REACT_APP_IS_COMMUNITIES_ENABLED && (
+        <>
+          <Typography variant="h2">{YOUR_COMMUNITIES_HEADING}</Typography>
+          <CommunitiesList />
+          <Button
+            onClick={() => setIsCommunitiesDialogOpen(true)}
+            className={classes.button}
+          >
+            {ALL_COMMUNITIES_LINK}
+          </Button>
+          <Dialog
+            aria-labelledby="communitiies-dialog-title"
+            open={isCommunitiesDialogOpen}
+            onClose={() => setIsCommunitiesDialogOpen(false)}
+          >
+            <DialogTitle id="communities-dialog-title">
+              {ALL_COMMUNITIES_HEADING}
+            </DialogTitle>
+            <DialogContent>
+              <CommunitiesList all />
+            </DialogContent>
+            <DialogActions>
+              <Button onClick={() => setIsCommunitiesDialogOpen(false)}>
+                {CLOSE}
+              </Button>
+            </DialogActions>
+          </Dialog>
+        </>
+      )}
+
+      <Markdown source={LANDING_TEXT} />
+
+      <Accordion className={classes.accordion}>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="weekly-events-content"
+          id="weekly-events-header"
+        >
+          <Typography variant="h2" className={classes.accordionTitle}>
+            {WEEKLY_EVENTS_TITLE}
+          </Typography>
+          <Typography className={classes.accordionSubtitle}>
+            {WEEKLY_EVENTS_SUBTITLE}
+          </Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Markdown source={WEEKLY_EVENTS_MARKDOWN} topHeaderLevel={3} />
+        </AccordionDetails>
+      </Accordion>
+
+      <Accordion className={classes.accordion}>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="updates-content"
+          id="updates-header"
+        >
+          <Typography variant="h2" className={classes.accordionTitle}>
+            {UPDATES_TITLE}
+          </Typography>
+          <Typography className={classes.accordionSubtitle}>
+            {LAST_UPDATE}
+          </Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Markdown source={UPDATES_MARKDOWN} topHeaderLevel={3} />
+        </AccordionDetails>
+      </Accordion>
+
+      <Accordion className={classes.accordion}>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="contribute-content"
+          id="contribute-header"
+        >
+          <Typography variant="h2" className={classes.accordionTitle}>
+            {CONTRIBUTE}
+          </Typography>
+          <Typography className={classes.accordionSubtitle}>
+            {JOIN_THE_TEAM}
+          </Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <ContributorForm />
```

Comment:
![Screenshot 2021-06-02 at 01 14 17](https://user-images.githubusercontent.com/13669362/120405172-e529a600-c33f-11eb-8b4b-7e81c654cc04.png)
The alignment of the button looks broken somehow it's now in the accordion...